tools/docker/installation.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Docker
+page: Installation
+section: tools
+---
+
+#**Getting started with Docker**
+
+###**Installing Docker on Fedora**
+
+####**Prerequisites**
+
+Docker is currently supported on the following versions of Fedora:
+
+* Fedora 20
+* Fedora 21
+* Fedora 22
+
+Docker requires a **64-bit** version of Fedora and a minimum kernel version of **3.10**. 
+
+The following command displays your kernel version and architecture:
+
+```
+$ uname -r
+4.0.8-300.fc22.x86_64
+```
+
+If your kernel version is less than 3.10, you can update it using the following command:
+
+```
+$ sudo dnf update kernel
```

Comment:
Good point. I think that currently only F21 and later are supported releases and these have (AFAIK) always shipped with kernel >= 3.10.